juju/client/overrides.py

```diff
@@ -275,10 +275,28 @@ def __repr__(self):
                                                 repr(self.source),
                                                 repr(self.value))
 
+
 class Resource(Type):
-    _toSchema = {'application': 'application', 'charmresource': 'CharmResource', 'id_': 'id', 'pending_id': 'pending-id', 'timestamp': 'timestamp', 'username': 'username', 'name': 'name', 'origin': 'origin'}
-    _toPy = {'CharmResource': 'charmresource', 'application': 'application', 'id': 'id_', 'pending-id': 'pending_id', 'timestamp': 'timestamp', 'username': 'username', 'name': 'name', 'origin': 'origin'}
-    def __init__(self, charmresource=None, application=None, id_=None, pending_id=None, timestamp=None, username=None, name=None, origin=None, **unknown_fields):
+    _toSchema = {'application': 'application',
```

Comment:
Is it worth a doc string here? I'm guessing the Resource class is heavily used?

Reply:
This is essentially an override of generated code.  At least some sort of comment to indicate what we're overriding and why makes sense.

Reply:
FWIW, the changes here are strictly for fixing lint errors.  Might have been better to just exclude this file, since it's mostly generated code; but it's also a mix of manually created code as well.

Reply:
On second thought, since this is just lint fixes, I'm going to punt on adding comments to this for now.